ass/src/main/java/cz/ackee/ass/activity/FeedbackActivity.kt

```diff
@@ -78,11 +80,10 @@ internal class FeedbackActivity : AppCompatActivity() {
             deviceModel = Build.MODEL,
             appVersion = packageInfo.versionName,
             deviceMake = Build.MANUFACTURER,
-            appName = applicationInfo.labelRes.let { resId ->
-                if (resId == 0) {
-                    applicationInfo.nonLocalizedLabel.toString()
-                } else {
-                    getString(resId)
+            appName = when {
```

Comment:
wouldnt it be better to move this to lazy above?

Reply:
✔️